### PR TITLE
[174] Calculate Product Shipping Cost

### PR DESCRIPTION
**Pull request by ToluEnny24**
I have calculated shipping cost

### CODE REVIEW

File: submissions/fabunmiTolulope/HTML/Productpage.html [hunk 1]
```diff
@@ -84,6 +85,14 @@ <h4>CONNECT WITH US</h4>
     <p>&copy; 2020 Greenprint Integrated Services.<br> Copyrights reserved.</p>
 	</div>
 </footer>
+<<<<<<< Updated upstream
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/HTML/Productpage.html:88]: You have a conflict here, kindly remove it

File: submissions/fabunmiTolulope/HTML/Productpage.html [hunk 2]
```diff
@@ -84,6 +85,14 @@ <h4>CONNECT WITH US</h4>
...
 <script src="../dom/newsletterForm.js"></script>
+=======
+<<<<<<< Updated upstream
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/HTML/Productpage.html:91]: Same here

File: submissions/fabunmiTolulope/HTML/Productpage.html [hunk 3]
```diff
@@ -84,6 +85,14 @@ <h4>CONNECT WITH US</h4>
...
+=======
+<script src="../dom/productPage.js"></script>
+<script src="../dom/newsletterForm.js"></script>
+>>>>>>> Stashed changes
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/HTML/Productpage.html:95]: And here too

File: submissions/fabunmiTolulope/dom/productPage.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        submit.disabled = true;
+        quantity.nextElementSibling.innerHTML = "Please enter a valid quantity";
+    } else{
+        submit.disabled = false;
+        quantity.nextElementSibling.innerHTML = "";
+    }
+}
+
+function priceValue() {
+        if (quantity.value > 0) {
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/dom/productPage.js:21]: Format this line of code

File: submissions/fabunmiTolulope/dom/productPage.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+function shippingAmount() {
+    if (productPrice < 1000) {
+        shippingCost.innerHTML = parseFloat(100);
+    } else if ((productPrice >= 1000) && (productPrice <= 20000)) {
+        shippingCost.innerHTML = parseFloat(0.15 * productPrice);
+    } else if ((productPrice >= 20000) && (productPrice <= 100000)) {
+        shippingCost.innerHTML = parseFloat(0.2 * productPrice);
+    } else if (productPrice >= 100000) {
+        shippingCost.innerHTML = parseFloat(25000);
+    } else {
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/dom/productPage.js:38]: this should return false

File: submissions/fabunmiTolulope/dom/productPage.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+        return true;
+    }
+}
+
+function totalCostValue() {
+    if (quantity.value > 0) {
+        totalCost.innerHTML = parseFloat(shippingCost.innerHTML) + parseFloat(productPrice);
+    } else{
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/dom/productPage.js:46]: this should return false

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/fabunmiTolulope/dom/productPage.js (hunk 4 above)
Comment (ToluEnny24) [reply submissions/fabunmiTolulope/dom/productPage.js:21]: Hi @adaugochi how do I format a line of code?

**Review by adaugochi — APPROVED** [review]

**Pull request merged**